tpm2/credactivation/credential_activation_test.go

```
@@ -0,0 +1,54 @@
+package credactivation
+
+import (
+	"bytes"
+	"crypto/rsa"
+	"encoding/base64"
+	"math/big"
+	insecureRand "math/rand"
+	"testing"
+
+	"github.com/google/go-tpm/tpm2"
+)
+
+func TestCredentialActivation(t *testing.T) {
+	// These values were independently tested/derived-from from TCG 2.0.38-compliant hardware.
+	n, ok := new(big.Int).SetString("21781359931719875035142348126986833104406251147281912291128410183893060751686286557235105177011038982931176491091366273712008774268043339103634631078508025847736699362996617038459342869130285665581223736549299195932345592253444537445668838861984376176364138265105552997914795970576284975601851753797509031880704132484924873723738272046545068767315124876824011679223652746414206246649323781826144832659865886735865286033208505363212876011411861316385696414905053502571926429826843117374014575605550176234010475825493066764152314323863950174296024693364113127191375694561947145403061250952175062770094723660429657392597", 10)
+	if !ok {
+		t.Fatalf("Failed to parse publicN string.")
+	}
+	public := rsa.PublicKey{
+		N: n,
+		E: 65537,
+	}
+
+	aikDigest, err := base64.StdEncoding.DecodeString("5snpf9qRfKD2Tb72eLAZqC/a/MyUhg+IvdwDZkTJK9w=")
```

write a `mustDecodeBase64` helper func to hide error handling